framework/decode/openxr_replay_consumer_base.cpp

```diff
@@ -823,8 +869,8 @@ void OpenXrReplayConsumerBase::Process_xrPollEvent(const ApiCallInfo&
     XrResult replay_result;
 
     // WIP: Put this constant somewhere interesting
-    const uint32_t kRetryLimit      = 10;
-    const int64_t  kMaxSleepLimitNs = 500000000; // 500ms
+    const uint32_t kRetryLimit      = 24;
+    const int64_t  kMaxSleepLimitNs = 50000000; // 50ms
```

Comment:
with the correct kRetryLimit the kMaxSleepLimitNs is redundant 1 << 24 is 16ms (for a total wait of 32ms)   (yes I know the two variables were originally my idea, but i was being too lazing to compute the total wait of 1 << (retry)  - 1 :)

Reply:
As I mentioned above, I just abandoned the sleep limit time because the retry limit is good enough.